css/style.css

```diff
@@ -88,7 +88,7 @@ nav {
   margin: 0 0 0 250px;
 }
 
-nav a {
+nav a:link, nav a:active {
```

Comment:
This `:link` addition breaks the style for when you visited the link. It looks like this for me, the not visible links have been visited.

![image](https://user-images.githubusercontent.com/282402/215586269-2bb27558-2b92-4b25-b92f-bc62613965b5.png)